### PR TITLE
Add textual clarification on -update and -select event names

### PR DESCRIPTION
**Pull request by EricOnFHIR**
https://jira.hl7.org/browse/FHIR-36792

### CODE REVIEW

**Comment by EricOnFHIR** [comment]
Also created DiagnosticReport for select profile since it was the last resource for which no profile existed.  Updated examples for DiagnosticReport update and select to ensure alignment with profiles.

File: input/pagecontent/3-6-3-DiagnosticReport-update.md [hunk 1]
```diff
@@ -14,10 +14,14 @@ The updates include:
...
-`study` | 0..1 | `ImagingStudy/{id}?_elements=identifier,accession` | Present if one or more attributes in the ImagingStudy resource associated with the report have changed
-`updates` | 1..1 | not applicable | Contains a single `Bundle` resource holding changes to be made to the current content of the [`anchor context`](5_glossary.html)
+`report` | 1..1 | `DiagnosticReport/{id}?_elements=identifier` | FHIR DiagnosticReport resource specifying the [`anchor context`](5_glossary.html) in which the update is being made.  Note that only the resource.resourceType and resource.id of the [`anchor context`](5_glossary.html) are required to be present.  Other attributes may be present in the DiagnosticReport resource if their values have changedsh  or were newly populated.
+`update` | 1..1 | Not Applicable | FHIR Bundle resource specifying the [`anchor context`](5_glossary.html) in which the update is being made.  Note that only the resource.resourceType and resource.id of the [`anchor context`](5_glossary.html) are required to be present.  Other attributes may be present in the DiagnosticReport resource if their values have changedsh  or were newly populated.
```
Comment (isaacvetter) [inline input/pagecontent/3-6-3-DiagnosticReport-update.md:18]:
```suggestion
`updates` | 1..1 | Not Applicable | FHIR Bundle resource specifying the [`anchor context`](5_glossary.html) in which the update is being made.  Note that only the resource.resourceType and resource.id of the [`anchor context`](5_glossary.html) are required to be present.  Other attributes may be present in the DiagnosticReport resource if their values have changedsh  or were newly populated.

```

Reverting to "update**s**", not "update". We think it was a typo.

File: input/pagecontent/3-6-3-DiagnosticReport-update.md [hunk 2]
```diff
@@ -14,10 +14,15 @@ The updates include:
...
-`patient` | 0..1 | `Patient/{id}?_elements=identifier` | Present if one or more attributes in the Patient resource associated with the report have changed.
-`study` | 0..1 | `ImagingStudy/{id}?_elements=identifier,accession` | Present if one or more attributes in the ImagingStudy resource associated with the report have changed
-`updates` | 1..1 | not applicable | Contains a single `Bundle` resource holding changes to be made to the current content of the [`anchor context`](5_glossary.html)
+`report` | 1..1 | `DiagnosticReport/{id}?_elements=identifier` | FHIR DiagnosticReport resource specifying the [`anchor context`](5_glossary.html) in which the update is being made.  Note that only the resource.resourceType and resource.id of the [`anchor context`](5_glossary.html) are required to be present.  Other attributes may be present in the DiagnosticReport resource if their values have changedsh  or were newly populated.
```
Comment (isaacvetter) [inline input/pagecontent/3-6-3-DiagnosticReport-update.md:17]:
```suggestion
`report` | 1..1 | `DiagnosticReport/{id}?_elements=identifier` | FHIR DiagnosticReport resource specifying the [`anchor context`](5_glossary.html) in which the update is being made.  Note that only the resource.resourceType and resource.id of the [`anchor context`](5_glossary.html) are required to be present.  Other attributes may be present in the DiagnosticReport resource if their values have changed or were newly populated.

```
fix typo

File: input/pagecontent/3-6-3-DiagnosticReport-update.md [hunk 3]
```diff
@@ -15,9 +15,15 @@ The updates include:
...
-`patient` | 0..1 | `Patient/{id}?_elements=identifier` | Present if one or more attributes in the Patient resource associated with the report have changed.
-`study` | 0..1 | `ImagingStudy/{id}?_elements=identifier,accession` | Present if one or more attributes in the ImagingStudy resource associated with the report have changed
-`updates` | 1..1 | not applicable | Contains a single `Bundle` resource holding changes to be made to the current content of the [`anchor context`](5_glossary.html)
+
+`updates` | 1..1 | Not Applicable | FHIR Bundle resource specifying the [`anchor context`](5_glossary.html) in which the update is being made.  Note that only the resource.resourceType and resource.id of the [`anchor context`](5_glossary.html) are required to be present.  Other attributes may be present in the DiagnosticReport resource if their values have changedsh  or were newly populated.
```
Comment (isaacvetter) [inline input/pagecontent/3-6-3-DiagnosticReport-update.md:19]:
```suggestion
`updates` | 1..1 | Not Applicable | FHIR Bundle resource specifying the [`anchor context`](5_glossary.html) in which the update is being made.  Note that only the resource.resourceType and resource.id of the [`anchor context`](5_glossary.html) are required to be present.  Other attributes may be present in the DiagnosticReport resource if their values have changed  or were newly populated.

```

File: input/pagecontent/3-6-3-DiagnosticReport-update.md [hunk 4]
```diff
@@ -15,9 +15,16 @@ The updates include:
 Key | Cardinality | FHIR operation to generate context | Description
 ----- | -------- | ---- | ---- 
 `report` | 1..1 | `DiagnosticReport/{id}?_elements=identifier` | FHIR DiagnosticReport resource specifying the [`anchor context`](5_glossary.html) in which the update is being made.  Note that only the resource.resourceType and resource.id of the [`anchor context`](5_glossary.html) are required to be present.  Other attributes may be present in the DiagnosticReport resource if their values have changed or were newly populated.
-`patient` | 0..1 | `Patient/{id}?_elements=identifier` | Present if one or more attributes in the Patient resource associated with the report have changed.
-`study` | 0..1 | `ImagingStudy/{id}?_elements=identifier,accession` | Present if one or more attributes in the ImagingStudy resource associated with the report have changed
-`updates` | 1..1 | not applicable | Contains a single `Bundle` resource holding changes to be made to the current content of the [`anchor context`](5_glossary.html)
+
```
Comment (isaacvetter) [inline input/pagecontent/3-6-3-DiagnosticReport-update.md:18]:
```suggestion

`patient` | 0..1 | `Patient/{id}?_elements=identifier` | Present if one or more attributes in the Patient resource associated with the report have changed.
`study` | 0..1 | `ImagingStudy/{id}?_elements=identifier,accession` | Present if one or more attributes in the ImagingStudy resource associated with the report have changed
```
Discussed on today's WG call, not making this change now. Need to better determine how content changes to the anchor resource are identified by -update recipients.

**Review by bvdh — APPROVED** [review]

File: input/pagecontent/2-3-Events.md [hunk 5]
```diff
@@ -117,37 +117,32 @@ This event category contains events required to maintain a FHIRcast session. The
...
 
-Note that any syncerrors resulting from select events pose significantly less risk than syncerrors resulting from other event types. 
+The `context` element in an update event SHALL contain at least two fields. One with the name of the `FHIRresource` which holds the anchor context and one named `updates` holding a single `Bundle` resource with entries holding the content being shared.  The `Bundle` resource SHALL conform to the [FHIRcast content update Bundle](StructureDefinition-fhircast-content-update-bundle.html) profile. 
 
-The most common events definitions have been provided in the [event catalog](3_Events.html). For an example see [`DiagnosticReport-select`](3-6-4-DiagnosticReport-select.html).
+FHIRcast supports all events that follow this format. The most common events definitions have been provided in the [event catalog](3_Events.html). For an example see [`DiagnosticReport-update`](3-6-3-DiagnosticReport-update.html).
 
-#### Content sharing events
+#### Selection events
```
Comment (isaacvetter) [inline input/pagecontent/2-3-Events.md:136]:
```suggestion
#### Selection events - Experimental

```

**Pull request merged**